src/paginationCalls/pageDescriptor.ts

```diff
@@ -72,7 +72,7 @@ export class PageDescriptor implements Descriptor {
     const maxResults = 'maxResults' in options ? options.maxResults : -1;
     let pushCount = 0;
     let started = false;
-    function callback(err: Error | null, resources, next) {
+    function callback(err: Error | null, resources: [], next: {}) {
```

Comment:
Is this syntax valid? I'd better write `resources: Array<{}>`

Reply:
this passes the type check, but I think `Array<{}>` makes better sense, changed it.